### PR TITLE
added tag normalizer

### PR DESCRIPTION
**Pull request by Sarahdeep**

### CODE REVIEW

File: ml-service/app/main.py [hunk 1]
```diff
@@ -2,6 +2,16 @@
 import torch
 from pydantic import BaseModel
 from own_lsg_converter import MYLSGConverter
+import uvicorn
```
Comment (OkDenAl) [inline ml-service/app/main.py:5]: а для чего его импортить?

Comment (Sarahdeep) [reply ml-service/app/main.py:5]: хз, было уже

Comment (OkDenAl) [reply ml-service/app/main.py:5]: удалишь плз? оно unused
